### PR TITLE
Connect to AWS RDS

### PR DESCRIPTION
**Pull request by yun-kwak**
Now we can use real DB.(MariaDB on AWS RDS)

### CODE REVIEW

File: server/src/database.js [hunk 1]
```diff
@@ -35,11 +64,14 @@ const createStore = () => {
         email: {
           type: Sequelize.STRING, allowNull: false,
         },
-        token: Sequelize.STRING,
         birthday: Sequelize.DATE,
         city: Sequelize.STRING,
         state: Sequelize.STRING,
+        zip: Sequelize.STRING,
+        additionalAddress: Sequelize.STRING,
```
Comment (ethkim) [inline server/src/database.js]: Should we just name this as address? Typically the address fields in US can be stored as:

- street address
- additional street address
- city
- state
- zip code

If we don't want to split into all these fields, we can just store everything into "address" field for now.

File: server/src/database.js [hunk 2]
```diff
@@ -35,11 +64,14 @@ const createStore = () => {
...
         phone: Sequelize.STRING,
+        description: Sequelize.STRING,
```
Comment (ethkim) [inline server/src/database.js]: What goes into description for User? Description is a poorly defined concept and all kinds of data can creep into this. Let's either rename to some more specific purpose, or remove this field.

File: server/src/database.js [hunk 3]
```diff
@@ -64,12 +96,80 @@ const createStore = () => {
     title: Sequelize.STRING,
     description: Sequelize.STRING,
     price: Sequelize.FLOAT,
-    maxParticipants: Sequelize.INTEGER,
+    type: Sequelize.INTEGER,
+    // 'type' specifies which type of the event is.
```
Comment (ethkim) [inline server/src/database.js]: Move the comment above the line the comment is describing (above 99).

File: server/src/database.js [hunk 4]
```diff
@@ -64,12 +96,80 @@ const createStore = () => {
...
+  Review.init({
+    userId: {
+      type: Sequelize.INTEGER,
+      primaryKey: true,
+      references: {
+        model: User,
+        key: 'id',
+      },
+    },
+    eventId: {
```
Comment (ethkim) [inline server/src/database.js]: This is for future thinking, and we can add TODOs to fix later.

Is a Review tied to an Event? Should that be tied to Schedule instead? Or should that be tied to the host ID? Some questions:
- Does the product ever need to indicate which schedule the user participated (i.e. "This is a review about the event held on June 15, 2019")?
- Can the host "delete" event or schedule? If that happens, where do all the reviews go? Reviews should always stay in the system (never deleted). 
- Take a look at how AirBnB handles reviews. There, all reviews are tied to the hosts regardless of different houses she has, or regardless of different bookings each house has. But that's how it's shown on UI - how can you implement this on the backend? Should reviews be tied to all 3 (hostID, eventID, scheduleID), and display the appropriate info at different product request?

File: server/src/database.js (hunk 1 above)
Comment (yun-kwak) [reply server/src/database.js]: Modified! Thank you.

**Pull request merged**